### DISCUSSION TITLE
Add Python 3.8 support

### DISCUSSION THREAD

**Discussion by hugovk** [community/community#25815] [Actions]
<p>Python 3.8.0 was <a href="https://discuss.python.org/t/python-3-8-0-is-now-available/2478" rel="nofollow noopener">released yesterday</a>, 2019-10-14:</p>
<ul>
<li><a href="https://discuss.python.org/t/python-3-8-0-is-now-available/2478" rel="nofollow noopener">https://discuss.python.org/t/python-3-8-0-is-now-available/2478</a></li>
</ul>
<p>Please could you also add support in GitHub Actions for Python 3.8?</p>
<p>Also requested at the action:</p>
<ul>
<li><a href="https://github.com/actions/setup-python/issues/30" rel="nofollow noopener">https://github.com/actions/setup-python/issues/30</a></li>
</ul>
<p>Thank you!</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams. Thanks again for reaching out :grinning:</p>

**Comment by cclauss** [comment]
<p>Python 3.8 is now available in GitHub Actions.</p>

**Comment by hugovk** [comment]
<p>Almost!</p>
<p>Python 3.8 is available for Ubuntu and Windows, but not macOS yet.</p>
<p><a href="https://github.com/actions/setup-python/issues/30#issuecomment-548732908" rel="nofollow noopener">https://github.com/actions/setup-python/issues/30#issuecomment-548732908</a></p>

**Comment by hugovk** [comment, marked as answer]
<p>Python 3.8 is now available on all three: Ubuntu, Windows and macOS.</p>
<p>Thanks!</p>
<p>(In second place of the CIs I’m checking. Travis CI: 21h from release to availability; GHA 21 days; Azure Pipelines and AppVeyor still WIP.)</p>
